wear/build.gradle

```diff
@@ -1,5 +1,39 @@
 apply plugin: 'com.android.application'
 
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath 'com.android.tools.build:gradle:2.3.1'
+    }
+}
+
+allprojects {
+    repositories {
+        jcenter()
+    }
+}
+
+project.ext{
```

Comment:
What I don't understand that you made this changes because (I suppose) your build failed, but I can build the project correctly.